### PR TITLE
feat: initial build of active directory docs

### PR DESCRIPTION
**Pull request by thezackm**
adding new docs to align with an upcoming release of an [integration to monitor active directory](https://github.com/newrelic/newrelic-active-directory-integration)

### CODE REVIEW

**Comment by nbaenam** [comment]
Hi @thezackm, it’d be great to know when this needs to be released, so we can plan accordingly.

**Comment by rhetoric101** [comment]
@thezackm it turns out that we can't use <code> tags anymore. Instead we use a sandwich of three backticks. I've converted your markdown tables to HTML tables, but I haven't yet been able to get a successful build yet.

Finally, could you sign the CLA again even though you've already done this? Our CLA job has a short memory!!

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/active-directory-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,668 @@
+---
+title: Active Directory integration
+tags:
+    - Active Directory integration
```
[the remark this answers is not in the document]
Comment (rhetoric101) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/active-directory-integration.mdx:4]: @zuluecho9 thanks for the tip. I'm going to give this a try.

**Comment by rhetoric101** [comment]
@thezackm I am closing this original PR since @homelessbirds is addressing this doc a new [PR](https://github.com/newrelic/docs-website/pull/13390). Sorry for the confusion about this PR!

**Review by homelessbirds — APPROVED** [review]

**Pull request merged**